recipes/doubtlab/meta.yaml

```diff
@@ -0,0 +1,154 @@
+{% set name = "doubtlab" %}
+{% set version = "0.1.1" %}
+
+
+##----------------------------------------------------------------------##
+## TODO for Maintainers:
+##
+## >> After each PyPI release of the package (doublab):
+## 
+##    1. [THIS GETS TAKEN CARE OF BY A BOT]: NO ACTION NEEDED
+##       Get the corresponding "version" and "sha256" of the latest 
+##       release and update the following (line number: 2 and 33).
+##       - version
+##       - source.sha256
+##
+##    2. [THIS IS A MANUAL PROCESS]
+##       Crosscheck and update the package dependency versions under:
+##       - requirements.host
+##       - requirements.run
+##
+## SEE THIS EXAMPLE: 
+## - how "pysteps" is maintaining "conda-forge/pysteps-feedstock":
+## https://pysteps.readthedocs.io/en/latest/developer_guide/update_conda_forge.html
+##
+##----------------------------------------------------------------------##
```

Comment:
These comments are not really necessary, right? Can we remove them?

Reply:
@ocefpaf I have removed/updated the comments to another file (`maintainers_notes.md`). This will serve as a single point of entry for sharing notes among the maintainers of this feedstock.

Reply:
Why do you need these notes? From what I can see, everything except for the TODOs is covered by the docs. Generally, we prefer to only use those and not add instructions in the recipe, if they're not recipe specific as it clutters the repo and might become outdated. If someone is not familiar with the process, the docs will always have the most up-to-date info.

The TODOs can be included as a single line under the respective items.

Reply:
Yep. Anything there that is not in our docs should be added to the docs, not to the recipe.

Reply:
Okay. I removed all the comments and the `maintainers_notes.md` file as well. I have kept all my notes in a GitHub gist. And perhaps maintaining them there is better, as in that case, a mere change in notes does not fire a new PR to the feedstock.